lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -50,6 +51,12 @@ def do_execute(machine, command, options={})
           end
         }
 
+        if options[:metainfo]
```

Comment:
Yes, during `build-origin-base`, etc, it's useful to see the command that failed since they don't get printed and there's quite a lot that goes on. I can defer to you, I don't feel super strongly on this.